### PR TITLE
SLCORE-316 Drop support of SonarQube < 7.9 for connected mode

### PR DESCRIPTION
**Pull request by damien-urruty-sonarsource**

### CODE REVIEW

File: its/tests/src/test/java/its/tools/ItUtils.java [hunk 1]
```diff
@@ -46,21 +46,20 @@
...
   static {
-    if ("LATEST_RELEASE[6.7]".equals(System.getProperty("sonar.runtimeVersion"))) {
+    if ("LATEST_RELEASE[7.9]".equals(System.getProperty("sonar.runtimeVersion"))) {
       Properties props = new Properties();
       try (Reader r = Files.newBufferedReader(Paths.get("../../core/src/main/resources/plugins_min_versions.txt"), StandardCharsets.UTF_8)) {
         props.load(r);
         javaVersion = props.getProperty("java");
         pythonVersion = props.getProperty("python");
         phpVersion = props.getProperty("php");
         // Can't test the version 4.0.0 since there was a change of custom rule API
```
Comment (henryju) [inline its/tests/src/test/java/its/tools/ItUtils.java:56]: Comment can be removed

File: its/tests/src/test/resources/java-sonarlint-package.xml [hunk 2]
```diff
@@ -16,7 +16,7 @@
       <priority>MAJOR</priority>
     </rule>
 
-    <!-- DEPRECATED keys for 6.7 axis -->
+    <!-- DEPRECATED keys for 7.9 axis -->
```
Comment (henryju) [inline its/tests/src/test/resources/java-sonarlint-package.xml:19]: Maybe we could change the comment to say this is to support SonarJava < 6.0 (https://jira.sonarsource.com/browse/SONARJAVA-3160)

File: its/tests/src/test/resources/java-sonarlint-with-hotspot.xml [hunk 3]
```diff
@@ -10,7 +10,7 @@
       <priority>MAJOR</priority>
     </rule>
     
-    <!-- DEPRECATED keys for 6.7 axis -->
+    <!-- DEPRECATED keys for 7.9 axis -->
```
Comment (henryju) [inline its/tests/src/test/resources/java-sonarlint-with-hotspot.xml:13]: Maybe we could change the comment to say this is to support SonarJava < 6.0 (https://jira.sonarsource.com/browse/SONARJAVA-3160)

File: its/tests/src/test/resources/java-sonarlint.xml [hunk 4]
```diff
@@ -22,7 +22,7 @@
       <priority>MAJOR</priority>
     </rule>
 
-    <!-- DEPRECATED keys for 6.7 axis -->
+    <!-- DEPRECATED keys for 7.9 axis -->
```
Comment (henryju) [inline its/tests/src/test/resources/java-sonarlint.xml:25]: Maybe we could change the comment to say this is to support SonarJava < 6.0 (https://jira.sonarsource.com/browse/SONARJAVA-3160)

**Review by henryju — CHANGES_REQUESTED** [review]
Additional place to update:
https://github.com/SonarSource/sonarlint-core/blob/c44e410a6bfddc446bb9d55615ac43801551d895/core/src/main/java/org/sonarsource/sonarlint/core/container/connected/update/PluginReferencesDownloader.java#L111

**Review by henryju — APPROVED** [review]

**Review by jblievremont — APPROVED** [review]

**Pull request merged**